examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -1143,8 +1157,24 @@ def compute_text_embeddings(prompt):
                         text_encoder_use_attention_mask=args.text_encoder_use_attention_mask,
                     )
 
+                if unet.config.in_channels > channels:
+                    needed_additional_channels = unet.config.in_channels - channels
+                    additional_latents = randn_tensor(
+                        (bsz, needed_additional_channels, height, width),
+                        device=noisy_model_input.device,
+                        dtype=noisy_model_input.dtype,
+                    )
+                    noisy_model_input = torch.cat([additional_latents, noisy_model_input], dim=1)
+
+                if args.class_labels_conditioning == "timesteps":
+                    class_labels = timesteps
+                else:
+                    class_labels = None
+
                 # Predict the noise residual
-                model_pred = unet(noisy_model_input, timesteps, encoder_hidden_states).sample
+                model_pred = unet(
+                    noisy_model_input, timesteps, encoder_hidden_states, class_labels=class_labels
+                ).sample
```

Comment:
IF stage II uses timesteps as "class_labels" conditioning. Note that class_labels is overloaded and this just means embeddings added with image latents

Reply:
Timesteps as class labels ...

That's confusing to me.

Reply:
Yeah I also think the name was badly chosen! Sorry think it was me who introduced it for another use case. The policy from now on is a bit that `class_labels` should always be integer inputs (either timesteps for super res noise, or actual embedding indexes - for e.g. sdx4 upscaler). So this is def ok here for me